ci/travis/install-bazel.sh

```diff
@@ -85,6 +83,7 @@ if [ "${TRAVIS-}" = true ] || [ -n "${GITHUB_WORKFLOW-}" ]; then
 # CI output doesn't scroll, so don't use curses
 build --color=yes
 build --curses=no
+build --disk_cache="$(test "${OSTYPE}" = msys || echo ~/ray-bazel-cache)"
```

Comment:
🧐what do we need the `test` condition?

Reply:
It's because I get random permission-denied errors on Windows that I haven't been able to figure out yet :\ I can look into it down the road but for now I'm just disabling the disk cache on Windows.

Reply:
didn't we have bazel cache before for windows?

Reply:
I've had `disk_cache` locally, but I don't think we ever had it on CI for Windows. The option previously was [just a few lines higher](https://github.com/ray-project/ray/pull/7988/files/a53b37ed9887ec1069562caa492a40a3420cd2ec..76c3150ca779ad98f9cf9425fa9564faf785f13d#diff-3b37d271915170b4a4a8a68fa2fb83f8L64) under `
if [ "${TRAVIS-}" = true ]; then ...`.

Reply:
It's likely due to the remote cache. We've always had that.